opal/mca/common/verbs/common_verbs_port.c

```diff
@@ -94,6 +94,10 @@ int opal_common_verbs_port_bw(struct ibv_port_attr *port_attr,
         /* 12x */
         *bandwidth *= 12;
         break;
+    case 16:
+        /* 12x */
+        *bandwidth *= 12;
```

Comment:
Are you sure you wanted the bandwidth to be *= 12 in this case?

Reply:
@gpaulsen Thanks for the catch. I confirmed that it should be 16 (UCX does the same https://github.com/openucx/ucx/blob/master/src/uct/ib/base/ib_iface.c#L1033)
Will make this change now.

Reply:
@Akshay-Venkatesh ^^^ ?